File: fe_public/quadra_mall_web-main/src/types/Order/orderRequest.ts
export interface OrderRequest {
    addressId: number; // ID địa chỉ giao hàng
    shippingMethod: string; // Phương thức vận chuyển
    paymentMethod: string; // Phương thức thanh toán
    voucherIds?: { [key: number]: number }; // { orderId: voucherId } - optional object
    orderIds: number[]; // Danh sách ID đơn hàng
    notes?: { [key: number]: string }; // { orderId: note } - optional object
}

File: be/quadramall_api-main/src/main/java/com/quadra/ecommerce_api/dto/custom/order/request/OrderRequest.java
package com.quadra.ecommerce_api.dto.custom.order.request;

import lombok.Data;

import java.lang.reflect.Array;
import java.util.ArrayList;
import java.util.List;
import java.util.Map;

@Data
public class OrderRequest {
    private List<Long> orderIds;
    private Long addressId;
    private String shippingMethod;
    private String paymentMethod;
    private Map<Long,Long> voucherIds;
    private Map<Long,String> notes;

}
